models/src/main/java/com/vimeo/networking2/common/Followable.kt

```diff
@@ -0,0 +1,26 @@
+package com.vimeo.networking2.common
+
+import com.vimeo.networking2.Metadata
+
+/**
+ * All DTOs that can be followed should conform this interface.
```

Comment:
This documentation is helpful for a developer of the library, but isn't particularly descriptive for a consumer of the library as it says how to implement the interface, but not how to use the interface. Perhaps rewording it to the following would be better for consumers:
```suggestion
 * Represents a data model that can be followed by the user. Provides access to a [Metadata] property that contains [FollowableInteractions].
```